docs_versioned_docs/version-ros2humble/ros/config/yaml/system.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: System
+sidebar_label: System
+sidebar_position: 3
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+Proper networking setup is crucial in setting up the ROS 2 middleware and ensure other onboard computers communicate reliably.
+
+## Hosts
+
+The **hosts** section serves as a way to match IP addresses to hostnames. By default, Clearpath robots use the serial number as the hostname and have a default IP of `192.168.131.1`.
+
+- The **self** entry refers to what hostname is this configuration file on.
```

Comment:
I do not understand this sentence. Should this say _...configuration file is saved to._ ?

Reply:
Changed it. Should say the hostname of the computer this configuration is loaded onto.